test/e2e/storage/testsuites/subpath.go

```diff
@@ -179,6 +179,11 @@ func (s *subPathTestSuite) defineTests(driver TestDriver, pattern testpatterns.T
 		testReadFile(f, l.filePathInVolume, l.pod, 1)
 	})
 
+	/*
```

Comment:
While the test is pretty straight forward the fact that it's a storage test may cause issues.
I would assume any cloud provider would have to support creating a directory and ensuring it's still there. :smile_cat: